filters/include/pcl/filters/impl/conditional_removal.hpp

```diff
@@ -530,49 +530,49 @@ pcl::PointDataAtOffset<PointT>::compare (const PointT& p, const double& val)
     case pcl::PCLPointField::INT8 :
     {
       std::int8_t pt_val;
-      memcpy (&pt_val, pt_data + this->offset_, sizeof (std::int8_t));
+      std::copy_n(pt_data + this->offset_, sizeof(std::int8_t), &pt_val);
```

Comment:
Is this correct?

Reply:
AFAIK:
```
template< class InputIt, class Size, class OutputIt >OutputIt copy_n( InputIt first, Size count, OutputIt result );
```

Reply:
[`memcpy`](https://en.cppreference.com/w/cpp/string/byte/memcpy) has a count of bytes, while copy_n has a count of objects. You seem to be mixing both

Reply:
Yeah, this is the sort of moving bytes into a byte array that `memcpy` is appropriate for. Will revert.